### PR TITLE
Case insensitive

### PR DESCRIPTION
**Pull request by NikitaPoly**
New usernames will all be lowercase while all old usernames set to be lowercase.

### CODE REVIEW

**Comment by luciipak** [comment]
Does it work for users logging in? like if they log in with a different case from what's in the db

File: deployments/apiv2/services/users/src/main.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ async def get_all_users(
     customer_id = uuid.UUID(hex=token["userid"])
 
     query = (
-        "SELECT id, name, email, created_at, last_login, suspended FROM users "
+        "SELECT id, LOWER(name), email, created_at, last_login, suspended FROM users "
```
Comment (luciipak) [inline deployments/apiv2/services/users/src/main.py:411]: Is the `LOWER` for extra safety measures? They should all be lowercase in the DB correct?

Comment (NikitaPoly) [reply deployments/apiv2/services/users/src/main.py:411]: I guess it is redundant. I will remove it

File: deployments/apiv2/services/users/src/main.py [hunk 2]
```diff
@@ -454,7 +454,7 @@ async def get_user(
...
```
Comment (luciipak) [inline deployments/apiv2/services/users/src/main.py:457]: this might also be redundant then?

**Comment by luciipak** [comment]
And does this work with all the tests?

**Comment by NikitaPoly** [comment]
All tests are good. Ran them again just to make sure

**Review by luciipak — APPROVED** [review]

**Pull request merged**